### ISSUE TITLE
Request for training script

### ISSUE THREAD

**Issue by Soester10** [saikatdutta/Stacked_DMSHN_bokeh#4]
Hello Sir,

While trying to replicate your work, I'm having some difficulties in training the model. The Adam optimizer is returning nan, and am not sure if its due to the wrong implementation of the loss function (tried implementing after reading your paper).  It would be greatly helpful if you could share your training script. You could send it to my email id: hnnhariharan12@gmail.com.

Thanks in advance.

**Comment by chenquan-hdu** [comment]
Hi，
I replicated stack-DMSHN a week ago, and it worked. I suggest you check your dataset and loss function.

**Comment by Soester10** [comment]
Hi,
Yes, thank you. I figured out where I went wrong and corrected it.
